{"is_source_file": true, "file_id": 2030, "git_revision": "", "state": 1, "description": "", "external_files": [], "external_methods": [], "published": [], "classes": [], "methods": [], "calls": [], "search-terms": [], "ctags": [{"_type": "tag", "name": "OEQADependency", "path": "/home/kavia/workspace/code-generation/Kirkstone/kirkstone/openembedded-core/meta/lib/oeqa/core/exception.py", "pattern": "/^class OEQADependency(OEQAException):$/", "language": "Python", "kind": "class"}, {"_type": "tag", "name": "OEQAException", "path": "/home/kavia/workspace/code-generation/Kirkstone/kirkstone/openembedded-core/meta/lib/oeqa/core/exception.py", "pattern": "/^class OEQAException(Exception):$/", "language": "Python", "kind": "class"}, {"_type": "tag", "name": "OEQAMissingManifest", "path": "/home/kavia/workspace/code-generation/Kirkstone/kirkstone/openembedded-core/meta/lib/oeqa/core/exception.py", "pattern": "/^class OEQAMissingManifest(OEQAException):$/", "language": "Python", "kind": "class"}, {"_type": "tag", "name": "OEQAMissingVariable", "path": "/home/kavia/workspace/code-generation/Kirkstone/kirkstone/openembedded-core/meta/lib/oeqa/core/exception.py", "pattern": "/^class OEQAMissingVariable(OEQAException):$/", "language": "Python", "kind": "class"}, {"_type": "tag", "name": "OEQAPreRun", "path": "/home/kavia/workspace/code-generation/Kirkstone/kirkstone/openembedded-core/meta/lib/oeqa/core/exception.py", "pattern": "/^class OEQAPreRun(OEQAException):$/", "language": "Python", "kind": "class"}, {"_type": "tag", "name": "OEQATestNotFound", "path": "/home/kavia/workspace/code-generation/Kirkstone/kirkstone/openembedded-core/meta/lib/oeqa/core/exception.py", "pattern": "/^class OEQATestNotFound(OEQAException):$/", "language": "Python", "kind": "class"}, {"_type": "tag", "name": "OEQATimeoutError", "path": "/home/kavia/workspace/code-generation/Kirkstone/kirkstone/openembedded-core/meta/lib/oeqa/core/exception.py", "pattern": "/^class OEQATimeoutError(OEQAException):$/", "language": "Python", "kind": "class"}], "format": "Python", "hash": "01493820310bf91d28f80d47477f40e5", "format-version": 4, "code-base-name": "meta-rdk-sampleapp", "filename": "openembedded-core/meta/lib/oeqa/core/exception.py", "knowledge_revision": 2030}